src/BLE_Client.cpp

```diff
@@ -303,7 +303,7 @@ bool SpinBLEClient::connectToServer()
             {
 
                 pRemoteCharacteristic->subscribe(true, notifyCallback);
-                if (pRemoteService->getUUID() == CYCLINGPOWERSERVICE_UUID)
+                if (myDevice == myPowerMeter)
```

Comment:
@doudar Can you explain these changes so I understand the net effect?

Reply:
Changed because as we keep adding more UUID's we're checking against, the line was getting long and I realized we already had a variable that always contained what we needed to check against for simplification.

Reply:
Mydevice being the device we're trying to connect and mypowermeter being the selected powermeter found during the last scan.

Reply:
Does the "selected" one handle "any"?

Reply:
"Any" or the name of the saved ble server is tested in the client code starting on line 476